rfc-parser/src/main/java/tech/pantheon/yanginator/parser/services/GrammarKitRFCUncomplaintUtils.java

```diff
@@ -780,4 +781,24 @@ private static List<String> adjustRelPathKeyexpr(List<String> lines) {
         }
         return result;
     }
+
+    /**
+     * When a decimal value was used, the integer-value statement was recognized first,
+     * and it resulted in an error.
+     * By swapping these statements, the decimal-value statement will be recognized first resulting
+     * in correctly identifying both decimal-value and integer-value
+     *
+     * @param lines list of strings
+     * @return list of strings
+     */
+    private static List<String> swapDecimalWithIntegerInRangeBoundaryDef(List<String> lines) {
+        List<String> result = new ArrayList<>();
+        for (String line : lines) {
+            if (line.contains("  integer-value | decimal-value")) {
```

Comment:
Is this really safe? what if this line contains for example "integer-value | decimal-value | string-value"?
Safer solution would be to look just for "integer-value | decimal-value" and then, instead of replacing whole line, replace just these statements, for example: line.replace("integer-value | decimal-value", "decimal-value | integer-value")